packages/plugin-expo-device/LICENSE.txt

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018 Bugsnag, https://www.bugsnag.com/
```

Comment:
It probably doesn't matter too much, but 2019?

Reply:
Good spot, thanks!